2025-09-09 03:14:52,284 INFO    MainThread:1558 [wandb_setup.py:_flush():81] Current SDK version is 0.21.3
2025-09-09 03:14:52,284 INFO    MainThread:1558 [wandb_setup.py:_flush():81] Configure stats pid to 1558
2025-09-09 03:14:52,284 INFO    MainThread:1558 [wandb_setup.py:_flush():81] Loading settings from /root/.config/wandb/settings
2025-09-09 03:14:52,284 INFO    MainThread:1558 [wandb_setup.py:_flush():81] Loading settings from /workspace/limx_rl/pointfoot-legged-gym/wandb/settings
2025-09-09 03:14:52,284 INFO    MainThread:1558 [wandb_setup.py:_flush():81] Loading settings from environment variables
2025-09-09 03:14:52,284 INFO    MainThread:1558 [wandb_init.py:setup_run_log_directory():686] Logging user logs to /workspace/limx_rl/pointfoot-legged-gym/logs/PF_TRON1A/wandb/run-20250909_031452-882figol/logs/debug.log
2025-09-09 03:14:52,284 INFO    MainThread:1558 [wandb_init.py:setup_run_log_directory():687] Logging internal logs to /workspace/limx_rl/pointfoot-legged-gym/logs/PF_TRON1A/wandb/run-20250909_031452-882figol/logs/debug-internal.log
2025-09-09 03:14:52,284 INFO    MainThread:1558 [wandb_init.py:init():813] calling init triggers
2025-09-09 03:14:52,284 INFO    MainThread:1558 [wandb_init.py:init():818] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-09-09 03:14:52,284 INFO    MainThread:1558 [wandb_init.py:init():854] starting backend
2025-09-09 03:14:52,561 INFO    MainThread:1558 [wandb_init.py:init():857] sending inform_init request
2025-09-09 03:14:52,589 INFO    MainThread:1558 [wandb_init.py:init():865] backend started and connected
2025-09-09 03:14:52,591 INFO    MainThread:1558 [wandb_init.py:init():936] updated telemetry
2025-09-09 03:14:52,592 INFO    MainThread:1558 [wandb_init.py:init():960] communicating run to backend with 90.0 second timeout
2025-09-09 03:14:53,119 INFO    MainThread:1558 [wandb_init.py:init():1011] starting run threads in backend
2025-09-09 03:14:53,181 INFO    MainThread:1558 [wandb_run.py:_console_start():2494] atexit reg
2025-09-09 03:14:53,181 INFO    MainThread:1558 [wandb_run.py:_redirect():2342] redirect: wrap_raw
2025-09-09 03:14:53,182 INFO    MainThread:1558 [wandb_run.py:_redirect():2411] Wrapping output streams.
2025-09-09 03:14:53,182 INFO    MainThread:1558 [wandb_run.py:_redirect():2434] Redirects installed.
2025-09-09 03:14:53,184 INFO    MainThread:1558 [wandb_init.py:init():1057] run started, returning control to user process
2025-09-09 03:14:53,184 INFO    MainThread:1558 [wandb_run.py:_config_callback():1380] config_cb None None {'runner_cfg': {'algorithm_class_name': 'PPO', 'checkpoint': -1, 'encoder_class_name': 'MLP_Encoder', 'experiment_name': 'PF_TRON1A', 'exptid': '', 'load_run': '-1', 'logger': 'both', 'max_iterations': 15000, 'num_steps_per_env': 24, 'policy_class_name': 'ActorCritic', 'resume': False, 'resume_path': 'None', 'run_name': '', 'save_interval': 500, 'wandb_project': 'IsaacGym_Tron'}}
2025-09-09 03:14:53,185 INFO    MainThread:1558 [wandb_run.py:_config_callback():1380] config_cb None None {'policy_cfg': {'activation': 'elu', 'actor_hidden_dims': [512, 256, 128], 'critic_hidden_dims': [512, 256, 128], 'init_noise_std': 1.0, 'orthogonal_init': False}}
2025-09-09 03:14:53,185 INFO    MainThread:1558 [wandb_run.py:_config_callback():1380] config_cb None None {'alg_cfg': {'clip_param': 0.2, 'critic_take_latent': True, 'desired_kl': 0.01, 'entropy_coef': 0.01, 'est_learning_rate': 0.001, 'gamma': 0.99, 'lam': 0.95, 'learning_rate': 0.001, 'max_grad_norm': 1.0, 'num_learning_epochs': 5, 'num_mini_batches': 4, 'schedule': 'adaptive', 'ts_learning_rate': 0.0001, 'use_clipped_value_loss': True, 'value_loss_coef': 1.0}}
2025-09-09 03:14:53,185 INFO    MainThread:1558 [wandb_run.py:_config_callback():1380] config_cb None None {'env_cfg': '<legged_gym.envs.pointfoot_flat.pointfoot_flat_config.BipedCfgPF object at 0x7b2feb4b0af0>'}
2025-09-09 03:18:26,055 INFO    wandb-AsyncioManager-main:1558 [service_client.py:_forward_responses():84] Reached EOF.
2025-09-09 03:18:26,055 INFO    wandb-AsyncioManager-main:1558 [mailbox.py:close():137] Closing mailbox, abandoning 1 handles.
2025-09-09 03:18:27,686 ERROR   wandb-AsyncioManager-main:1558 [asyncio_manager.py:fn_wrap_exceptions():183] Uncaught exception in run_soon callback.
Traceback (most recent call last):
  File "/opt/conda/envs/pointfoot_legged_gym/lib/python3.8/site-packages/wandb/sdk/lib/asyncio_manager.py", line 181, in fn_wrap_exceptions
    await fn()
  File "/opt/conda/envs/pointfoot_legged_gym/lib/python3.8/site-packages/wandb/sdk/lib/service/service_client.py", line 65, in _send_server_request
    await self._writer.drain()
  File "/opt/conda/envs/pointfoot_legged_gym/lib/python3.8/asyncio/streams.py", line 387, in drain
    await self._protocol._drain_helper()
  File "/opt/conda/envs/pointfoot_legged_gym/lib/python3.8/asyncio/streams.py", line 190, in _drain_helper
    raise ConnectionResetError('Connection lost')
ConnectionResetError: Connection lost
